microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -39,11 +39,29 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
+
+Follow the steps below to create custom applications that you can use to connect to Microsoft Graph PowerShell. Use this approach if you need to limit the consent permissions granted for Graph PowerShell usage.
+
+1. Go to the <a href="https://entra.microsoft.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade" target="_blank">Azure portal - App registrations</a> > **New Registration**
+    1. **Name**: *Microsoft Graph PowerShell - High Privilege admin use only* (Enter a meaningful name for your application)
+    1. **Supported account types**: Accounts in this organization directory
+    1. **Redirect URI**:
+        - Select **Public client/native** from the drop down
+        - **Uri**: http://localhost
+        - Click **Create**
+
```

Comment:
@msewaweru The Create button appears to have been renamed to Register in the UI. Need to update the instruction above to say Click Register instead of Click Create.

We also need to add these steps to make the process complete.

1. Select **Overview** blade and click **Go to Enterprise applications**
1. Select the **Properties** blade and set **Assignment required?** to **Yes**
1. Click **Save**
1. Select the **Users and groups** blade and add the users and groups permitted to use this Graph PowerShell app.